### ISSUE TITLE
ancient icesim4-compat release for IceTop (Trac #2065)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2057]
Hi,

A while ago I made a branch of the releases to be used in IceTop simulations. We discussed these things a lot back then, but they somehow never ended in the branches directory of icerec. These are the metaprojects in question:
http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/sandbox/jgonzalez/iceprod/meta-projects/IC2011-L2_V12-08-00_IceSim4compat_V4-IT
http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/sandbox/jgonzalez/iceprod/meta-projects/IC2012-L2_V13-01-00_IceSim4compat_V3-IT

Would it be possible to copy them to the icerec branches area so they do not reside in my sandbox?

Migrated from https://code.icecube.wisc.edu/ticket/2065
```json
{
    "status": "closed", 
    "changetime": "2017-08-03T11:40:37", 
    "description": "Hi,\n\nA while ago I made a branch of the releases to be used in IceTop simulations. We discussed these things a lot back then, but they somehow never ended in the branches directory of icerec. These are the metaprojects in question:\nhttp://code.icecube.wisc.edu/projects/icecube/browser/IceCube/sandbox/jgonzalez/iceprod/meta-projects/IC2011-L2_V12-08-00_IceSim4compat_V4-IT\nhttp://code.icecube.wisc.edu/projects/icecube/browser/IceCube/sandbox/jgonzalez/iceprod/meta-projects/IC2012-L2_V13-01-00_IceSim4compat_V3-IT\n\nWould it be possible to copy them to the icerec branches area so they do not reside in my sandbox?", 
    "reporter": "jgonzalez", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1501760437018025", 
    "component": "combo reconstruction", 
    "summary": "ancient icesim4-compat release for IceTop", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-08-02T15:27:29", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170802T19:43:45: 
* david.schultz changed status from "new" to "assigned"
* david.schultz commented:

> Or preferably, make them releases.  Since these are used in production.


* david.schultz changed owner from "" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20170803T11:38:16: kjmeagher commented:

> In changeset 157497:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157497"
> move production meta-projects to releases from sandbox see #2057
> ```

**Comment by icecube-trac** [comment]
Trac update at 20170803T11:40:37: 
* kjmeagher changed status from "assigned" to "closed"
* kjmeagher commented:

> In changeset 157498:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157498"
> move production meta-projects to releases from sandbox fixes #2057
> ```


* kjmeagher changed resolution from "" to "fixed"
